pkg/cloudevents/observability/observer.go

```diff
@@ -27,12 +27,11 @@ type Reporter interface {
 }
 
 type reporter struct {
-	ctx     context.Context
-	span    *trace.Span
-	on      Observable
-	start   time.Time
-	measure stats.Measure
```

Comment:
`measure stats.Measure` - I am suppressed this is not used. I think we need the measurement.

Reply:
ah, looks like it was a migration leftovers. 👍